src/lib.rs

```diff
@@ -115,6 +115,8 @@ pub trait HasContext {
 
     unsafe fn create_texture(&self) -> Result<Self::Texture, String>;
 
+    unsafe fn create_textures(&self, target: u32) -> Result<Self::Texture, String>;
```

Comment:
These will also needed to be added to web_sys.rs (https://github.com/grovesNL/glow/blob/main/src/web_sys.rs) because the web backend implements this `HasContext` trait.

If you're not sure how to implement them correctly following the other examples in that file, we can leave them as `unimplemented!()` and I can fill the out afterwards

Reply:
I am not at all familiar with webgl, so i will go with `unimplemented!()`